sigproxy/main.go

```diff
@@ -0,0 +1,72 @@
+package main
+
+import (
+	"log"
+	"os"
+	"os/exec"
+	"os/signal"
+	"syscall"
+)
+
+func checkArguments() {
+	if len(os.Args) == 1 {
+		log.Fatal("USAGE: sigproxy <command> [arguments ...]")
+	}
+}
+
+func installSignalHandler() chan os.Signal {
+	sc := make(chan os.Signal, 1)
+	signal.Notify(sc, os.Interrupt)
+
+	return sc
+}
+
+func execCommand() *exec.Cmd {
+	// First argument is command to run, remainder are its arguments
+	cmd := exec.Command(os.Args[1], os.Args[2:]...)
+
+	// These conveniently default to /dev/null otherwise...
+	cmd.Stdin = os.Stdin
+	cmd.Stdout = os.Stdout
+	cmd.Stderr = os.Stderr
+
+	if err := cmd.Start(); err != nil {
+		log.Fatal(err)
+	}
+
```

Comment:
I don't think it would hurt to close them, and it is perhaps more correct (keep stderr open though, just in case). But I think it would be rare that it makes a difference in practice, and, depending on the details of how docker handles stdout/stderr, it might make no difference at all.